app/assets/stylesheets/components/common/_download_link.scss

```diff
@@ -0,0 +1,11 @@
+// For links that contain download attribute
+
+a[download] {
+  &:after {
+    @extend %icon;
+    @extend .icon--pdf;
```

Comment:
Very good point! Classes are a good solution for this, ill take a look, thanks @slaywell